6-AdvancedScenarios/3-call-api-acrs/SPA/src/pages/TodoList.jsx

```diff
@@ -39,16 +39,10 @@ const TodoListContent = () => {
  * https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-react/docs/getting-started.md
  */
 export const TodoList = () => {
```

Comment:
Why is this removed?

Reply:
It was causing an automatic sign-in every time the user signs out